reproduce-figures/prepare-metadata-files.R

```diff
@@ -0,0 +1,58 @@
+# This script can be used to create versions of `scpca-sample-metadata.tsv` and `scpca-library-metadata.tsv` needed to
+# reproduce manuscript figures, tables, and analyses.
+#
+# To run this script, you will first need to download all sample data from the ScPCA Portal: <https://scpca.alexslemonade.org/>
+# From this page, click the `Get All Sample Metadata` button on the top-right of the page.
+# Provide this TSV file as input to this script as:
+#
+# Rscript prepare-metadata-files.R --portal_sample_metadata <path to metadata.tsv>
+#
+# Files will be exported to the directory the script is run from, but this can be specified with `--output_dir`:
+#
+# Rscript prepare-metadata-files.R --portal_sample_metadata <path to metadata.tsv> --output_dir <path to output directory>
+
+renv::load()
+library(optparse)
+
+# Parse options --------
+option_list <- list(
+  make_option(
+    "--portal_sample_metadata",
+    type = "character",
+    help = "Path to TSV of all sample metadata downloaded from ScPCA Portal"
+  ),
+  make_option(
+    "--output_dir",
+    type = "character",
+    default = ".",
```

Comment:
I think we might actually want to make a default folder for this to keep things organized. Maybe `modified-portal-metadata`?

Reply:
What about just `metadata-files`? I suggest that specifically because it matches the script name.